drawee/src/main/java/com/facebook/drawee/drawable/ScalingUtils.java

```diff
@@ -193,6 +199,19 @@ public static Matrix getTransform(
         transform.postTranslate((int) (dx + 0.5f), (int) (dy + 0.5f));
         break;
 
+      case FIT_Y:
+        scale = scaleY;
+
+        float localScaleX = scaleY;
+        if (childWidth * scale < parentWidth) {
+          localScaleX = scaleX;
+        }
+
+        dx = parentBounds.left + (parentWidth - childWidth * localScaleX) * 0.5f;
+        transform.setScale(localScaleX, scale);
```

Comment:
This is incorrect. In order to preserve the aspect ratio, image must be scaled uniformly by x and y.

```
transform.setScale(scale, scale);
```